doc/docs.md

```diff
@@ -3150,6 +3150,17 @@ pub fn say_hi() {
 	println('hello from mymodule!')
 }
 ```
+All items inside a module can be used between the files of a module regardless of whether or
+not they're prefaced with the `pub` keyword.
+```v
```

Comment:
Use `v ignore` or `v failcompile` here, otherwise `v check-md doc/docs.md` will report an error and the CI will fail (it is checking each code snippet independently from all other).

Reply:
I knew that was gonna cause a problem! I saw `v failcompile` on other code blocks. But some just said `v`, so I wasn't sure which one to use. Noted for future self 🫡